CHANGES.md

```diff
@@ -1,5 +1,32 @@
 # Changes
 
+## edge-23.5.1
+
+This edge release introduces the ability to configure the proxy's discovery cache
+timeouts via annotations. While most users will not need to do this, it can be
+useful to improve the mesh's resilience to control plane failures. This release
+also includes a number of other important improvements and bug fixes.
+
+* Added -o json flag for the `linkerd multicluster gateways` command (thanks
+  @hiteshwani29)
+* Added missing label `linkerd.io/extension` to certain resources to ensure they
+  pruned when appropriate (thanks @Clement)
+* Fixed a memory leak in the service mirror controller
+* Improved validation of the `--to` and `--from` flags for the `linkerd viz stat`
+  command (thanks @pranoyk)
+* Fixed an issue with W3C trace context propagation which caused proxy spans to
+  be siblings rather than children of their original parent (thanks
+  @whiskeysierra)
+* Fixed a regression in the Linkerd CNI plugin where the CNI init script failed
+  without the `bash` executable
```

Comment:
i believe this bug was introduced as part of the change to switch the base image from Debian to Alpine, which was not included in the previous release, so the regression was never actually visible to users. Instead, let's maybe change this bullet point to just state that we changed the base image from Debian to Alpine, which is the user-facing change in the CNI plugin.